Fabulous.XamarinForms/samples/AllControls/AllControls/AllControls.fs

```diff
@@ -736,37 +752,37 @@ module App =
+                                .RelativeLayoutYConstraint(Constraint.RelativeToParent(fun parent -> parent.Height / 2.0))
                           MainPageButton
-                                .XConstraint(Constraint.RelativeToParent(fun parent -> parent.Width / 2.0))
+                                .RelativeLayoutXConstraint(Constraint.RelativeToParent(fun parent -> parent.Width / 2.0))
                       ])))
 
 
                dependsOn model.Count (fun model count -> 
                    View.ContentPage(title="AbsoluteLayout", useSafeArea=true,
-                       padding = new Thickness (10.0, 20.0, 10.0, 5.0), 
+                       padding = Thickness (10.0, 20.0, 10.0, 5.0), 
                        content= View.StackLayout(
                            children=[ 
-                               View.Label(text = "AbsoluteLayout Demo", fontSize = Device.GetNamedSize(NamedSize.Large, typeof<Label>), horizontalOptions = LayoutOptions.Center)
+                               View.Label(text = "AbsoluteLayout Demo", fontSize = Named NamedSize.Large, horizontalOptions = LayoutOptions.Center)
                                View.AbsoluteLayout(backgroundColor = Color.Blue.WithLuminosity(0.9), 
                                    verticalOptions = LayoutOptions.FillAndExpand, 
                                    children = [
                                       View.Label(text = "Top Left", textColor = Color.Black)
-                                          .LayoutFlags(AbsoluteLayoutFlags.PositionProportional)
-                                          .LayoutBounds(Rectangle(0.0, 0.0, AbsoluteLayout.AutoSize, AbsoluteLayout.AutoSize))
+                                          .AbsoluteLayoutLayoutFlags(AbsoluteLayoutFlags.PositionProportional)
+                                          .AbsoluteLayoutLayoutBounds(Rectangle(0.0, 0.0, AbsoluteLayout.AutoSize, AbsoluteLayout.AutoSize))
                                       View.Label(text = "Centered", textColor = Color.Black)
-                                          .LayoutFlags(AbsoluteLayoutFlags.PositionProportional)
-                                          .LayoutBounds(Rectangle(0.5, 0.5, AbsoluteLayout.AutoSize, AbsoluteLayout.AutoSize))
+                                          .AbsoluteLayoutLayoutFlags(AbsoluteLayoutFlags.PositionProportional)
+                                          .AbsoluteLayoutLayoutBounds(Rectangle(0.5, 0.5, AbsoluteLayout.AutoSize, AbsoluteLayout.AutoSize))
                                       View.Label(text = "Bottom Right", textColor = Color.Black)
-                                          .LayoutFlags(AbsoluteLayoutFlags.PositionProportional)
-                                          .LayoutBounds(Rectangle(1.0, 1.0, AbsoluteLayout.AutoSize, AbsoluteLayout.AutoSize)) ])
+                                          .AbsoluteLayoutLayoutFlags(AbsoluteLayoutFlags.PositionProportional)
```

Comment:
Shame about the long names, I guess they are unavoidable?

Reply:
I changed the way AttributeKeys are de-duplicated (with their corresponding methods like this one). Names are simplified only if a property (same name, same signature) is used on 2+ controls.

But after updating the samples, I realize it isn't a good idea since it makes those methods name longer for nothing (and breaks existing code).

So I will revert to the previous behavior instead. (making names shorter)

Cases where name collision with different signatures might happen, but hopefully they'll be rare. (in this case, it will be using unique names)